bentoml/_internal/frameworks/FRAMEWORK_TEMPLATE_PY

```diff
@@ -167,10 +167,11 @@ def get_runnable(bento_model: bentoml.Model) -> t.Type[bentoml.Runnable]:
     Private API: use :obj:`~bentoml.Model.to_runnable` instead.
     """
 
-    class FrameworkRunnable(bentoml.Runnable):
-        SUPPORT_NVIDIA_GPU = True  # type: ignore
-        SUPPORT_CPU_MULTI_THREADING = True  # type: ignore
-
+    class FrameworkRunnable(
+        bentoml.Runnable,
+        supported_resources={"nvidia.com/gpu"},
+        supports_multi_threading=True
```

Comment:
as mentioned before, even GPUs have a multi_threading concept, too.
still suggest using "support_cpu_multi_threading" to be clear

Reply:
I think imo `supports_multi_threading` is nicer UI, where we this is use in conjunction with `support_resources`

Reply:
We're using `SUPPORTS_CPU_MULTI_THREADING`.